docs/apache-airflow/concepts/operators.rst

```diff
@@ -103,6 +103,8 @@ You can also use Jinja templating with nested fields, as long as these nested fi
         dag=dag,
     )
 
+    t = transform_data([MyDataReader("/tmp/{{ ds }}/my_file")])
```

Comment:
Should this replace the PythonOperator above instead?

Reply:
yes @uranusjr I didn't remove the PythonOperator part. I will remove it.